__tests__/app.test.js

```diff
@@ -0,0 +1,54 @@
+const app = require("../app")
+const seed = require("../db/seeds/seed")
+const request = require('supertest')
+const db = require("../db/connection")
+const testData = require("../db/data/test-data/index")
+
+
+beforeEach(() => {
+    return seed(testData)
+    });
+
+  afterAll(() => {
+    db.end()
+  });
+
+describe("GET API/topics", () => {
+    test("Should return an array", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({body}) => {
+            expect(body).toBeInstanceOf(Array)
+        })
+    })
+    test("Array should have objects inside of it.", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({body}) => {
+            expect(body[0]).toBeInstanceOf(Object)
+        })
+    })
+    test("Objects in the array should have the predefined keys", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({body}) => {
+           body.map((treasure) => {
```

Comment:
A forEach would be more appropriate here, as we aren't interested in returning/creating a new array, rather we just want to iterate through each element and make an assertion.